### PR TITLE
Move `classpathOptions` to `CompileConfiguration`

### PR DESCRIPTION
**Pull request by Duhemm**
`classpathOptions` used to be a parameter of `AnalyzingCompiler`. I
believe, however, that it makes more sense to have it as an input of
compilation (like the `scalacOptions`) rather than as a property of
`AnalyzingCompiler`.

### CODE REVIEW

File: internal/compiler-interface/src/main/java/xsbti/ArtifactInfo.java [hunk 1]
```diff
@@ -15,11 +15,11 @@
 	public static final String ScalaOrganization = "org.scala-lang";
 
 	/** Define the name used to identify the sbt organization. */
-	public static final String SbtOrganization = "org.scala-sbt";
+	public static final String SbtOrganization = "ch.epfl.scala";
```
Comment (jvican) [inline internal/compiler-interface/src/main/java/xsbti/ArtifactInfo.java:18]: Why is this required? :smile:

**Review by jvican — APPROVED** [review]
LGTM! :+1:

File: internal/compiler-interface/src/main/java/xsbti/ArtifactInfo.java (hunk 1 above)
Comment (Duhemm) [reply internal/compiler-interface/src/main/java/xsbti/ArtifactInfo.java:18]: Otherwise the tests won't find the bridge's sources 😢

Comment (jvican) [reply internal/compiler-interface/src/main/java/xsbti/ArtifactInfo.java:18]: This will need to be removed when we try to merge upstream, then :wink:

**Comment by jvican** [comment]
It would be great if when you've got some time you can PR this upstream @Duhemm, I can imagine people being interested in this change there.
